README.md

```diff
@@ -96,6 +96,7 @@ Here is an incomplete list of some projects that started off with this template:
 - [beaker-py](https://github.com/allenai/beaker-py)
 - [gantry](https://github.com/allenai/beaker-gantry)
 - [ip-bot](https://github.com/abe-101/ip-bot)
+- [atyt](https://github.com/mstuttgart/atty)
```

Comment:
typo?
```suggestion
- [atty](https://github.com/mstuttgart/atty)
```

Reply:
Nop.  It's `atty`: https://github.com/mstuttgart/atty

Reply:
right... so, not "atyt"?

Reply:
> right... so, not "atyt"?

nop. It's `atty` from `a` lacri `tty` (alacritty terminal: https://github.com/alacritty)

Reply:
@epwalsh sorry. my fault hahah . Thanks to review.